dts/bindings/net/wireless/generic-coex-gpio.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022-2023 Dronetag s.r.o.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Generic representation of Coexistance pin interface for radios. This
+    interface is usually available on Wifi/Bluetooth/LTE modules to
+    interact with each other when sharing same antenna. This prevents
+    any collisions between transmissions from different modules. The grant
+    signal should signal that the external transceiver/module is not
+    transmitting. Therefore you are free to perform any TX operations as
+    required. When grant pin becomes inactive then you have time to
+    finish all of the ongoing TX operations before the external
+    transceiver begins their transmission. This is specified by the
+    grant-delay-us property.
+
+compatible: "generic-coex-gpio"
```

Comment:
I'd probably name this `gpio-radio-coex`, following "gpio-leds" naming,

ref https://docs.zephyrproject.org/latest/build/dts/bindings-syntax.html#compatible

Reply:
+1

Reply:
I was following the generic-fem-two-ctrl-pins.yaml example by Nordic.
Should we address the issue or is it blocking for merge ?

Reply:
Changed the compatible to gpio-radio-coex in 2e7ece47945aaa589983aa02c99bfed61744d7d2